analysis/analysis.go

```diff
@@ -38,59 +37,70 @@ func ReceiveRequest(c echo.Context) error {
 	repository := types.Repository{}
 	err := c.Bind(&repository)
 	if err != nil {
-		return c.JSON(http.StatusBadRequest, map[string]string{"result": "error", "details": "Error binding repository."})
+		log.Warning("ReceiveRequest", "ANALYSIS", 101, err)
```

Comment:
It seems that this an error message instead of a warning. Don't you think? I mean, if it generated an HTTP response of bad request, it was because an error occurred. Right?

Reply:
Thank's for that, @carlosljr ! As `repository` is an user input (JSON), I am suggesting using `Warning` because in this case will not be "HuskyCI error" itself but an user one. 

If a malicious user sends multiples requests to this route (fuzzing tools, for example), `glbgelf` will send multiples "101 Warnings" and not "Errors". 

What path to do you think I must follow? Error or Warning? 😄

Reply:
Ok, @rafaveira3. I understand that flood attacks may occur in this case. However, with warning messages, I think that is a kind of generalization. It was not necessarily generated from an attack. To indicate warning messages for a possible attack, you need to identify that multiple requests is coming with the same error in a short time period, and a warning loglevel with a different message could be generated. In this case, I think is more appropriate multiple error messages being generated from multiple requests. The same way, you could identify a possible attack attempt from those message logs. The message is more related with treatment of the user input (an error from the user) generated in Husky.

You can get a second opinion. If you prefer to keep warning messages, it's ok! It wouldn't impact the pull request approval.

Reply:
Thank's for all these awesome info! Sounds perfectly correct! I have turned these logs into errors and fixed some minor errors in this commit 22c4f68.